src/webgpu/api/validation/createTexture.spec.ts

```diff
@@ -372,15 +373,20 @@ g.test('sampleCount,valid_sampleCount_with_other_parameter_varies')
       usage,
     };
 
+    const usableWithStorageUsage =
+      (usage & GPUConst.TextureUsage.STORAGE_BINDING) === 0 ||
+      isTextureFormatUsableAsStorageFormat(format, t.isCompatibility);
+
     const success =
-      sampleCount === 1 ||
-      (sampleCount === 4 &&
-        (dimension === '2d' || dimension === undefined) &&
-        kTextureFormatInfo[format].multisample &&
-        mipLevelCount === 1 &&
-        arrayLayerCount === 1 &&
-        (usage & GPUConst.TextureUsage.RENDER_ATTACHMENT) !== 0 &&
-        (usage & GPUConst.TextureUsage.STORAGE_BINDING) === 0);
+      usableWithStorageUsage &&
+      (sampleCount === 1 ||
```

Comment:
nit: Since `sampleCount === 4` already has `(usage & GPUConst.TextureUsage.STORAGE_BINDING) === 0`
Maybe just group the newFlag `usableWithStorageUsage` with `(sampleCount === 1 && usableWithStorageUsage) || ...`

Reply:
done